dask/dataframe/io/csv.py

```diff
@@ -609,7 +613,8 @@ def to_csv(
     **kwargs
 ):
     """
-    Store Dask DataFrame to CSV files
+    Store Dask DataFrame to either a single CSV file or a group of CSV
+    files.
```

Comment:
I would prefer not to advertise the single file feature as prominently as this.  It is likely to perform much worse.  Instead, I would prefer that we keep the feature limited to just the keyword description below, and revert this change to the docstring title.